tools/EventClients/lib/java/src/org/xbmc/eventclient/XBMCClient.java

```diff
@@ -61,9 +61,12 @@ public XBMCClient(InetAddress hostAddress, int hostPort, String deviceName, Stri
 			iconType = Packet.ICON_GIF;
 
 		// Read the icon file to the byte array...
-		FileInputStream iconFileStream = new FileInputStream(iconFile);
-		byte[] iconData = new byte[iconFileStream.available()];
-		iconFileStream.read(iconData);
+		byte[] iconData;
+    	// Read the icon file to the byte array...
+    	try (FileInputStream iconFileStream = new FileInputStream(iconFile)) {
+        	iconData = new byte[iconFileStream.available()];
+        	iconFileStream.read(iconData);
+    	}
```

Comment:
indentation is off now

Reply:
Thanks a lot. I'm using sublime and my plugins messed this up. I'll fix.

Reply:
still wrong, please apply the patch jenkins posted.